akka-projection-testkit/src/main/scala/akka/projection/testkit/javadsl/ProjectionTestKit.scala

```diff
@@ -95,28 +97,32 @@ final class ProjectionTestKit private[akka] (testKit: ActorTestKit) {
         .run()(testKit.system.classicSystem)
 
     try {
-      probe.awaitAssert(max, interval, () => {
-        assertFunction.run()
-        Done
-      })
+      probe.awaitAssert(max, interval, () => assertFunction.create())
     } finally {
       Await.result(running.stop(), max.asScala)
     }
   }
 
   /**
-   * Run a Projection with an attached `TestSink` allowing
-   * control over the pace the elements flow through the Projection.
+   * Run a Projection with an attached `TestSubscriber.Probe` allowing
+   * control over the pace in which the elements flow through the Projection.
    *
-   * The Projection starts as soon as the first element is requested by the `TestSink`, new elements will be emitted
-   * as requested by the `TestSink`. The Projection won't stop by itself, therefore it's recommended to cancel the
-   * `TestSink` probe to gracefully stop the Projection.
+   * The assertion function receives a `TestSubscriber.Probe` that you can use
+   * request elements.
+   *
+   * The Projection starts as soon as the first element is requested by the `TestSubscriber.Probe`, new elements will be emitted
+   * as requested. The Projection is stopped once the assert function completes.
```

Comment:
incomplete sentence "will as requested"

Reply:
??

The sentence ends with: "new elements will be emitted as requested"